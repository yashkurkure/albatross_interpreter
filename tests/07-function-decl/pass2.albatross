var global int := 10;

fun function int () {
  return 1*2;
}

return global+3;
