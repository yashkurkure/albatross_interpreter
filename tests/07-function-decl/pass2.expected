Variable declared "global" type int
Function declared "function" returns int
Variable read "global" type int
